requirements-test.txt

```diff
@@ -1,11 +1,12 @@
-black
-coverage
-flake8<3.9.0,>=3.8.0
-hacking
-mock>=1.0
-pytest
-pytest-cov
-pep8>=1.6.0,<1.7
-pylint>=2.5.2,<2.6
+# 202404: Align all requirements with st2 test-requirements
+coverage==7.2.7
+black==22.3.0
+flake8==7.0.0
+mock==4.0.3
+pytest==6.2.5
+pytest-cov==4.1.0
+pep8==1.7.1
+pylint==3.1.0
 twine
-unittest2
+# 202404: Use forked version for flake8 v7.0.0 to align requirements with st2 test-requirements
+hacking @ git+https://github.com/nzlosh/hacking@flake8v7
```

Comment:
What is `hacking` used for?

Reply:
Oh. I see it in the old requirements.
Could you submit your branch as an upstream PR?

Reply:
I can.  In the mid-term I think we should drop `hacking`completely.

Reply:
What value does `hacking` provide? I'm inclined to just disable it.

Reply:
It provides a set of openstack coding style checks.  https://docs.openstack.org/hacking/latest/user/hacking.html#styleguide. I assume that these guidelines are used in St2 as a result of its heritage.

Reply:
`hacking` is only used for the orquesta repo. Looking through the list of things it enforces, I don't like this one: 
- [H301] Do not import more than one module per line (*)

Reply:
But whatever. It's not worth the hassle of moving it to another org. Approved.